README.md

```diff
@@ -26,339 +26,211 @@
-  and batching algorithms that combine individual inference requests
-  together to improve inference throughput. These scheduling and
-  batching decisions are transparent to the client requesting
-  inference.
-
-* [Extensible
-  backends](https://github.com/triton-inference-server/backend). In
-  addition to deep-learning frameworks, Triton provides a *backend
-  API* that allows Triton to be extended with any model execution
-  logic implemented in
-  [Python](https://github.com/triton-inference-server/python_backend)
-  or
-  [C++](https://github.com/triton-inference-server/backend/blob/main/README.md#triton-backend-api),
-  while still benefiting from full CPU and GPU support, concurrent
-  execution, dynamic batching and other features provided by Triton.
-
-* Model pipelines using
+under-development progress towards the next release. The current release is 
+version [2.21.0](https://github.com/triton-inference-server/server/tree/r22.04)
+and corresponds to the 22.04 container release on 
+[NVIDIA GPU Cloud (NGC)](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tritonserver).**
+
+----
+Triton Inference Server is an inference serving software that streamlines AI 
+inferencing. Triton enables teams to deploy any AI model from multiple deep 
+learning and machine learning frameworks, including TensorRT, TensorFlow, PyTorch, 
+ONNX, OpenVINO, Python and more. Triton supports inference across cloud, data center,
+edge and embedded devices on NVIDIA GPUs, x86 and ARM CPU, or AWS Inferentia. 
+Triton delivers optimized performance for many query types, including real time,
+batched, ensembles and audio/video streaming.
+
+Major features include:
+
+- [Multiple deep learning
+  framework support](https://github.com/triton-inference-server/backend).
+- [Multiple machine learning
+  framework support](https://github.com/triton-inference-server/fil_backend).
+- [Concurrent model
+  execution](docs/architecture.md#concurrent-model-execution).
+- [Dynamic batching](docs/architecture.md#models-and-schedulers).
+- Provides [Extensible
+  backend](https://github.com/triton-inference-server/backend) with a *backend
+  API*.
+- Model pipelines using
   [Ensembling](docs/architecture.md#ensemble-models) or [Business
   Logic Scripting
   (BLS)](https://github.com/triton-inference-server/python_backend#business-logic-scripting).
-  A Triton *ensemble* represents a pipeline of one or more models and
-  the connection of input and output tensors between those
-  models. *BLS* allows a pipeline along with extra business logic to
-  be represented in Python. In both cases a single inference request
-  will trigger the execution of the entire pipeline.
-
-* [HTTP/REST and GRPC inference
+- [HTTP/REST and GRPC inference
   protocols](docs/inference_protocols.md) based on the community
   developed [KServe
   protocol](https://github.com/kserve/kserve/tree/master/docs/predict-api/v2).
+- A [C API](docs/inference_protocols.md#in-process-triton-server-api) and
```

Comment:
Should be "in-process API", and then have C and Java "flavor"

Reply:
Unless there a link specifically for C API, I will leave it. This is also how it was in the original README. We can call it out below.